knotx-core/knotx-template-engine/src/test/java/com/cognifide/knotx/engine/placeholders/UriTransformerPlaceholderTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Knot.x - Reactive microservice assembler - Templating Engine Verticle
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.engine.placeholders;
+
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UriTransformerPlaceholderTest {
+
+    @Test
+    public void testGetPlaceholders() {
```

Comment:
Non-descriptive test method name. Please describe the context and expected result.